### PR TITLE
Update docs version. Fixes #1491

### LINKED ISSUE #1491: Update docs with Marathon releases
Looks like the docs still point to the 0.8.0 release: https://mesosphere.github.io/marathon/

### PR DESCRIPTION
**Pull request by ssk2**

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Marathon packages are available from Mesosphere's [repositories](http://mesosphe
 
 1.  Download and unpack the latest Marathon release.
 
-    **For Mesos 0.20.0:**
```
Comment (ConnorDoyle) [inline docs/docs/index.md]: Want to change this to `0.20.0+`?

Comment (ssk2) [inline docs/docs/index.md]: Done!

**Comment by kolloch** [comment]
Hi @ssk2 ,

I like the 0.20.0+ but as I explained to you in the email, we do not want to update the download page right now because of the upcoming 0.8.2 release this week. Thus we will merge this to master but not yet to the gh-pages branch.

**Comment by ssk2** [comment]
@kolloch - makes sense, thanks!

**Pull request merged**